externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/session/TrinoSessionImpl.scala

```diff
@@ -109,6 +109,19 @@ class TrinoSessionImpl(
     super.runOperation(operation)
   }
 
+  override def getInfo(infoType: TGetInfoType): TGetInfoValue = withAcquireRelease() {
+    infoType match {
+      case TGetInfoType.CLI_SERVER_NAME | TGetInfoType.CLI_DBMS_NAME =>
+        TGetInfoValue.stringValue("Trino")
+      case TGetInfoType.CLI_DBMS_VER => TGetInfoValue.stringValue(TRINO_COMPILE_VERSION)
```

Comment:
Can we use server version?
Ref: https://github.com/trinodb/trino/blob/f4726d67e102dbd43bbd70a2e89bfb27e4a21203/client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDatabaseMetaData.java#L140-L148